### ISSUE TITLE
Discussion: What else should we implement from serlo.org?

### ISSUE THREAD

**Issue by kulla** [serlo/frontend#64]
In this issue I list the features which are currently missing in the frontend project but which are implemented at serlo.org

- [ ] Support for Hotjar (we might want to discuss this issue)
- [ ] Popup for the email newsletter https://github.com/serlo/serlo.org/blob/6983e369dff10dadd6b3832bf9be489f4619435d/packages/public/server/src/module/Ui/templates/layout/partials/popup.phtml
- [ ] The following MathJax extensions have been enabled. Do we need them?
     - [ ] [mediawiki-texvc](http://docs.mathjax.org/en/latest/input/tex/extensions/mediawiki-texvc.html)
     - [ ] [mhchem](http://docs.mathjax.org/en/latest/input/tex/extensions/mhchem.html)

**Comment by Entkenntnis** [comment]
The cookie consent banner is alos missing?

**Comment by kulla** [comment]
> The cookie consent banner is missing as well?

Yes, that's right :-)

**Comment by kulla** [comment]
What I need to check:

- [ ] https://github.com/serlo/serlo.org/tree/master/packages/public/client
- [ ] https://github.com/serlo/serlo.org/tree/master/packages/public/server/src/module/Ui/templates/layout

**Comment by Entkenntnis** [comment]
see #65

**Comment by inyono** [comment]
- [ ] Sentry

**Comment by inyono** [comment]
- [ ] Add custom tag to Google Analytics so that we can filter between old & redesign

**Comment by elbotho** [comment]
@inyono could you elaborate or point to a person that can implement that? :)

**Comment by elbotho** [comment]
`<meta robot` probably not useful. [See here](https://webmasters.stackexchange.com/questions/48153/what-does-all-mean-in-meta-robots-tag)
I opted to add our robots.txt but not include this <meta tag

**Comment by elbotho** [comment]
favicon can also be omitted if it's in the default location. (memo to self: check if we should switch to PNG and offer bigger sizes as well)

**Comment by elbotho** [comment]
Moved opensearch to backlog (#88) because it would need changes in the current search implementation.

**Comment by elbotho** [comment]
**Content Types:**

`<meta name="content_type" content="article">`
I'm working on a rough implementation, but we probably need some normalizing.

these should be the strings we need to match:
`course, course-page, topic-folder, text-exercise-group, text-exercise, curriculum-topic-folder, applet
topic, curriculum-topic, topic-folder …`

**Comment by Entkenntnis** [comment]
As we move away from MathJax, th extensions will not be implemented, but if the features are needed probably with a matching KaTex-Extension.

Followup-Issues for the other two points: #180 #181
